controller/handler/dashboard.js

```diff
@@ -15,41 +15,83 @@ const database = require('../../model');
  * @returns {View} Rendered page
  */
 function dashboardView (request, reply) {
-    const trial = database.sequelize.model('trial');
+    const todayDate = moment();
```

Comment:
@poojaRal: Sure. I will change it to currentDate. 
@ChristianMurphy: I used it to maintain consistency in code and promote usage of moment.js in future. :grinning: If you don't want it, I can use new Date()